lib/ImageGallery/ImageGallery.js

```diff
@@ -0,0 +1,279 @@
+          </>
+        )}
+        {isBestSeller && (
+          <span
+            className={clsx(
+              'absolute px-2 py-1 text-base font-bold text-white uppercase bg-gray-400 top-2 left-2',
+              classNames.bestSellerTag,
+            )}
+          >
+            best seller
+          </span>
+        )}
+      </div>
+      <div className="relative w-fit h-fit">
+        <div
+          id="thumnails-reel"
+          ref={scrollableReel}
+          onMouseDown={dragStart}
+          onMouseMove={handleDrag}
+          className={clsx(
+            'flex shrink-0 scrollbar-hide',
+            {
+              'flex-col ml-1 space-y-1 lg:ml-0 lg:mr-4 lg:space-y-2 overflow-y-auto overflow-x-hidden h-56 md:h-80 lg:h-120':
+                isThumbnailsSide,
+              'space-x-1 mt-1 lg:space-x-2 lg:mt-4 overflow-x-auto overflow-y-hidden w-56 md:w-80 lg:w-104':
+                isThumbnailsBottom,
+              'cursor-grabbing': dragging,
+            },
+            classNames.thumbnailsReel,
+          )}
+        >
+          {images.map((image, index) => (
+            <div
+              key={image.imageUrl}
+              onClick={() => handleThumbnailChange(index)}
+              className={clsx(
+                'select-none w-[50px] h-[50px] md:w-[75px] md:h-[75px] lg:w-[100px] lg:h-[100px] shrink-0 bg-gray-900',
+                {
+                  'cursor-grabbing': dragging,
+                  'cursor-pointer': !dragging,
+                },
+                classNames.thumbnailWrapper,
+              )}
+              {...(navigateOnHover && {
+                onMouseEnter: () => debouncedOnHoverNavigation(index),
+              })}
+            >
+              {image.Image ? (
+                <image.Image />
+              ) : (
+                <img
+                  src={image.imageUrl}
+                  alt={image.alt}
+                  draggable={false}
+                  className={clsx(
+                    'object-contain w-full h-full',
+                    classNames.thumbnailImage,
+                  )}
+                />
+              )}
+            </div>
+          ))}
+        </div>
+        {!scrollForward && isReelScrollable && (
+          <button
+            className={clsx(
+              'absolute flex items-center justify-center w-6 h-6 bg-black/90',
+              {
+                'top-0 left-1/2 -translate-x-3 lg:-translate-x-5 rotate-90':
+                  isThumbnailsSide,
+                'left-0 -translate-y-1/2 top-1/2': isThumbnailsBottom,
+              },
+            )}
+          >
+            <LeftChevron
+              className="w-3 h-4 stroke-white"
+              viewBox="0 0 16 40"
+              onClick={moveReelBackWard}
+            />
+          </button>
+        )}
+        {scrollForward && isReelScrollable && (
+          <button
+            className={clsx(
+              'absolute flex items-center justify-center w-6 h-6 bg-black/90 mx-auto',
```

Comment:
I see we're using a lot of single quotes around here, but the standard for this project is double quotes. Can you run a quick `npx prettier -w .`?

Reply:
@loan-laux looking at the project I see single quotes for function's parameters

Reply:
That is true, sorry I got a bit confused. Thanks for pointing it out!